docs/contents/components/forms/file-input/index.ja.mdx

```diff
@@ -136,10 +163,16 @@ import { FileInput } from "@yamada-ui/react"
 
 ```tsx
 <VStack>
-  <FileInput isReadOnly variant="outline" placeholder="outline" />
-  <FileInput isReadOnly variant="filled" placeholder="filled" />
-  <FileInput isReadOnly variant="flushed" placeholder="flushed" />
-  <FileInput isReadOnly variant="unstyled" placeholder="unstyled" />
+  <For each={["outline", "filled", "flushed", "unstyled"]}>
+    {(variant, index) => (
+      <FileInput
+        key={index}
+        isReadOnly
```

Comment:
The props with `is*` will be deprecated in #3866. Therefore, it might be a good idea to update the documentation to remove `is*`. However, the actual removal hasn't been implemented yet, which could lead to conflicts.

I think it would be fine to update the documentation to remove `is*` in a separate PR. (Though, honestly, it feels a bit bothersome... :<)

Reply:
@mei28 
The modification regarding the `is` prefix in this PR does not seem appropriate. It would be better to address the `is` prefix in a different PR.